bundle/edu.gemini.model.p1/src/main/scala/edu/gemini/model/p1/overheads/Overheads.scala

```diff
@@ -97,7 +97,7 @@ object Overheads extends (BlueprintBase => Option[Overheads]) {
     case _: GnirsBlueprintImaging       => SimpleOverheads(0.10, 10, 0.193).some
     case _: GnirsBlueprintSpectroscopy  => SimpleOverheads(0.25, 15, 0.080).some
 
-    case _: NifsBlueprint               => SimpleOverheads(0.25, 11, 0.175).some
+    case _: NifsBlueprintBase           => SimpleOverheads(0.25, 11, 0.175).some
```

Comment:
I wonder if it would be more clear to add two cases here rather than matching the superclass. I don't feel strongly about it though. This match is very brittle, sadly.